<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Features_Normal</class>
 <widget class="QWidget" name="Features_Normal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Normal Estimation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_MaxDepthChangeFactor">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.020000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_Compute">
        <property name="text">
         <string>Compute Normal</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_Radius">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.020000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_Radius_Search">
        <property name="text">
         <string>Radius Search</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioButton_ImageIntegral">
        <property name="text">
         <string>ImageIntegral</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_NormalSmoothingSize">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_K_Search">
        <property name="text">
         <string>K Search</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox_K">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_Comput_All_Normals">
        <property name="text">
         <string>Compute All Normals</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radioButton_Radius_Search</tabstop>
  <tabstop>radioButton_K_Search</tabstop>
  <tabstop>radioButton_ImageIntegral</tabstop>
  <tabstop>doubleSpinBox_Radius</tabstop>
  <tabstop>spinBox_K</tabstop>
  <tabstop>doubleSpinBox_MaxDepthChangeFactor</tabstop>
  <tabstop>doubleSpinBox_NormalSmoothingSize</tabstop>
  <tabstop>pushButton_Compute</tabstop>
  <tabstop>pushButton_Comput_All_Normals</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
